File: task1.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ConsoleApp1
{
    internal class Program
    {
        static void Main(string[] args)
        {
            //Task-1: Ədədlərdən ibarət olan, verilmiş arrayin ədədi ortasını tapın.
            double[] ourArray = {1,2,3};
            double sum = 0;
            for (int i = 0; i < ourArray.Length; i++)
            {
                sum += ourArray[i];
            }
            Console.WriteLine(sum/ourArray.Length);
        }
    }
}

File: task1.js
//Task-1:Ədədlərdən ibarət olan, verilmiş arrayin ədədi ortasını tapın.

let ourArray =[1,2,3,4];
let sum = 0;

for(let i=0; i<ourArray.length; i++){
    sum+=ourArray[i];
}
console.log(sum/ourArray.length);